examples/sw/in.mos2

```diff
@@ -0,0 +1,33 @@
+# bilayer MoS2
+units      	metal
+boundary   	p p f
+processors	* * 1
+
+atom_style 	full
+read_data       ./single_layer_MoS2.data
```

Comment:
Please remove the "./" for increased portability

Reply:
Fixed.